karyon2-admin/src/main/java/netflix/admin/AdminConfigImpl.java

```diff
@@ -85,4 +97,28 @@ public boolean shouldScanClassPathForPluginDiscovery() {
     public int listenPort() {
         return listenPort;
     }
+
+    @Override
+    public List<Filter> additionalFilters() {
+        if (rootContextFilters.isEmpty()) {
+            new ArrayList<>();
+        }
+
+        List<Filter> filters = new ArrayList<>();
+        final String[] filterClasses = rootContextFilters.split(",");
+        for (String filterClass : filterClasses) {
+            try {
+                final Class<?> filterCls = Class.forName(filterClass);
```

Comment:
May want to call Class.forName(filterClass, false, classLoader) to avoid static init if we don't actually create an instance of this class.